includes/definitions/hpe-mapdu.yaml

```diff
@@ -0,0 +1,26 @@
+os: hpe-mapdu
+text: 'HPE Managed Power Distribution Unit'
+type: power
+icon: hpe
+group: hpe
+over:
+    - {graph: device_power, text: Power}  
+    - {graph: device_current, text: Current}
+mib_dir: hp
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.232.165.7
+            - .1.3.6.1.4.1.232.165.11
+snmp_max_oid: '8'
```

Comment:
This is why your detection is failing it should be:
`snmp_max_oid: 8`

Reply:
I have got some errors during unit tests about string expectancy on that parameter - will see.
About poller modules, understand and removed.
Thanks